src/components/ShowAllServices/ShowAllServices.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import {serviceService as serviceAPI} from '../../services/service/service.service';
+import ServiceList from '../../containers/ServiceList/ServiceList';
+import {Redirect} from 'react-router';
+import { toast } from 'react-toastify';
+import { withTranslation } from 'react-i18next';
+
+class ShowAllServices extends Component {
+	constructor(props){
+		super(props);
+		this.state = {
+			services: null,
+			redirect: false
+		};
+		this.redirect = this.redirect.bind(this);
+	}
+
+	componentDidMount(){
+		serviceAPI.getAll()
```

Comment:
This should be serviceService, not serviceAPI to be consistent with the other files.